cumulus_library/actions/exporter.py

```diff
@@ -82,5 +81,5 @@ def export_study(
         rich.print("The following tables were empty and were not exported:")
         for table in skipped_tables:
             rich.print(f"  - {table}")
-    if archive:
-        base_utils.zip_dir(path, data_path, manifest.get_study_prefix())
+    manifest.write_manifest(data_path)
```

Comment:
This solves the 'well this namespace should really be used for uploading instead' problem by just reusing it verbatim, which i think is a nice property.

This would mean in the future, for metadata, we'd just have a new section in the manifest defining what the path to metadata files are. Still TBD what that is at this point.